samples/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client/Pages/Home.razor

```diff
@@ -0,0 +1,132 @@
+                    <li>Handles client credentials flow</li>
+                    <li>Caches tokens to local file</li>
+                    <li>Manages token expiration</li>
+                </ul>
+
+                <p><strong>AuthenticationManager.cs/.ts</strong></p>
+                <ul>
+                    <li>C# wrapper around TypeScript implementation</li>
+                    <li>Integrates with ArcGIS Maps SDK for JavaScript</li>
+                    <li>Registers tokens with ArcGIS Identity Manager</li>
+                    <li>Provides authentication state management</li>
+                </ul>
+            </div>
+
+            <div class="alert alert-primary">
+                <strong>Ready to Test?</strong>
+                <p class="mb-2">Choose which render mode demonstration you'd like to see:</p>
+                <div class="row">
+                    <div class="col-md-6">
+                        <div class="card border-primary">
+                            <div class="card-body">
+                                <h6 class="card-title">🚀 InteractiveAuto Mode</h6>
+                                <p class="card-text small">Starts with Server-side rendering, then switches to WebAssembly. Best for initial load performance.</p>
+                                <a href="/map-auto" class="btn btn-primary btn-sm">View Auto Mode Demo</a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-6">
+                        <div class="card border-success">
+                            <div class="card-body">
+                                <h6 class="card-title">💻 InteractiveWebAssembly Mode</h6>
+                                <p class="card-text small">Pure client-side execution. Best for rich interactivity and offline capability.</p>
+                                <a href="/map-webassembly" class="btn btn-success btn-sm">View WebAssembly Demo</a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            
+            <div>
+                <p>
+                    Once your tokens are configured and working, navigate to the <strong>Map</strong> page 
+                    to see the fully authenticated GeoBlazor map in action! 
+                </p>
+                <ul>
+                    <li>Automatic token refresh keeps your map sessions active</li>
+                    <li>Secure server-side authentication protects your credentials</li>
+                    <li>Seamless integration with ArcGIS Portal or ArcGIS Online</li>
+                </ul>
+                <p>
+                    Ready to learn more? Check out the <a href="https://docs.geoblazor.com" target="_blank">GeoBlazor documentation</a> 
+                    and <a href="https://samples.geoblazor.com" target="_blank">samples</a> to unlock more possibilities!
```

Comment:
This is too generic at the bottom. Anyone using this sample is already a GB user. I would remove it or point to something more specifically relevant.